src/components/radioGroup/radio/radio.tsx

```diff
@@ -0,0 +1,64 @@
+import * as RadioGroup from '@radix-ui/react-radio-group';
+import classNames from 'classnames';
+import type React from 'react';
+import { useId, type ComponentPropsWithoutRef } from 'react';
+import { Icon, IconType } from '../../icon';
+
+export interface IRadioProps extends ComponentPropsWithoutRef<'button'> {
```

Comment:
I would probably extend the `RadixGroup.Item` properties as well here

Reply:
I don't disagree per se, but in turn we must find a way to document props without directing people to Radix.

Reply:
That's actually a good point, we could update the documentation for such components to say something like: "The component is based on the radix-ui radio implementation [here](link), check the available properties there" or something like that. I'd suggest to create a ticket to check how we can best approach this and update the components props/docs accordingly. What do you think?

Reply:
Yeah, lets investigate because the closer the documentation is, the better for the consumer. Sucks having to open multiple pages to figure out how a singular component works.